x-pack/plugins/cloud_security_posture/server/routes/status/status.ts

```diff
@@ -96,8 +119,28 @@ const getCspStatus = async ({
   const latestCspPackageVersion = latestCspPackage.version;
 
   const MIN_DATE = 0;
+  const indicesStatus = {
+    findingsLatestIndexStatus,
+    findingsIndexStatus,
+    scoreIndexStatus,
+  };
```

Comment:
_nit_
we grouped these to an object but then used them individually in `indicesDetails`

consider inlining this object directly to its consumer